src/styles.scss

```
@@ -1,5 +1,6 @@
 @use '@angular/material' as mat;
-@include mat.core();
+@include mat.elevation-classes();
```

Stylelint (17.4.0)

[error] Unexpected parentheses in argumentless mixin "mat.elevation-classes" call (scss/at-mixin-argumentless-call-parentheses)

(scss/at-mixin-argumentless-call-parentheses)